java/org/contikios/cooja/motes/AbstractApplicationMote.java

```diff
@@ -46,7 +46,7 @@
  *
  * @author Fredrik Osterlind
  */
-public abstract class AbstractApplicationMote extends AbstractWakeupMote implements Mote {
+public abstract class AbstractApplicationMote extends AbstractWakeupMote<MoteType,SectionMoteMemory> implements Mote {
```

Comment:
Spaces after the commas like in AbstractWakeupMote?

Reply:
Fixed.